pkg/agent/containerd/containerd.go

```diff
@@ -122,5 +127,41 @@ func Run(ctx context.Context, cfg *config.Node) error {
 		}
 	}
 
+	imageDir := "/var/lib/rancher/k3s/agent/images"
```

Comment:
You need to add a new struct field, `cfg.Containerd.Images` which is the path to the images and then setup the real path in pkg/agent/config/config.go run(...)